### PR TITLE
feat(nuxt, schema): add `<NoScript>` component and `noscript` typings

### PR DESCRIPTION
**Pull request by vis97c**
### 🔗 Linked issue

#4646 

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Following @danielroe & @pi0 recommendations I'm adding typings for the meta feature noscript

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: packages/schema/src/config/_app.ts [hunk 1]
```diff
@@ -191,7 +195,8 @@ export default {
     meta: [],
     link: [],
     style: [],
-    script: []
+    script: [],
+    noscript: []
```
Comment (danielroe) [inline packages/schema/src/config/_app.ts:199]: Would you move this up to `app.head`? `meta` is deprecated and only here for backwards compatibility.

File: docs/content/3.api/1.composables/use-head.md [hunk 2]
```diff
@@ -43,5 +43,6 @@ export default {
 * **link**: array, each item maps to a newly-created `<link>` element, where object properties map to attributes.
 * **style**: array, each item maps to a newly-created `<style>` element, where object properties map to attributes.
 * **script**: array, each item maps to a newly-created `<script>` element, where object properties map to attributes.
+* **noscript**: array, each item maps to a newly-created `<noscript>` element, where object properties map to
```
Comment (danielroe) [inline docs/content/3.api/1.composables/use-head.md:46]:
```suggestion
* **noscript**: array, each item maps to a newly-created `<noscript>` element, where object properties map to attributes
```

File: packages/schema/src/config/_app.ts (hunk 1 above)
Comment (danielroe) [inline packages/schema/src/config/_app.ts:199]: Would you move this up to `app.head`? `meta` is deprecated and only here for backwards compatibility.
```suggestion
    script: []
```

File: docs/content/3.api/1.composables/use-head.md [hunk 3]
```diff
@@ -43,5 +43,6 @@ export default {
 * **link**: array, each item maps to a newly-created `<link>` element, where object properties map to attributes.
 * **style**: array, each item maps to a newly-created `<style>` element, where object properties map to attributes.
 * **script**: array, each item maps to a newly-created `<script>` element, where object properties map to attributes.
+* **noscript**: array, each item maps to a newly-created `<noscript>` element, where object properties map to attributes
```
Comment (danielroe) [inline docs/content/3.api/1.composables/use-head.md:46]:
```suggestion
* **noscript**: array, each item maps to a newly-created `<noscript>` element, where object properties map to attributes.
```

File: packages/nuxt/src/head/runtime/components.ts [hunk 4]
```diff
@@ -86,6 +86,29 @@ export const Script = defineComponent({
   }))
 })
 
+// <script>
```
Comment (pi0) [inline packages/nuxt/src/head/runtime/components.ts:89]:
```suggestion
// <noscript>
```

**Comment by vis97c** [comment]
Oh wow didn't expected you to do the fixes. It was already late so i just wanted to send the pr and fix it later. Thanks @pi0 @danielroe

**Pull request merged**